ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -233,46 +233,129 @@ class SignalProducerSpec: QuickSpec {
 		}
 
 		describe("first") {
-			pending("should start a signal then block on the first value") {
+			it("should start a signal then block on the first value") {
+				var sink: Signal<Int, NoError>.Observer!
+				let producer = SignalProducer<Int, NoError> { observer, _ in
+					sink = observer
+				}
+				expect(sink).to(beNil())
+
+				var result: Result<Int, NoError>?
+				dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0)) {
```

Comment:
This can be removed with the [use of `buffer`](https://github.com/ReactiveCocoa/ReactiveCocoa/pull/1710/files#r24038113).